### PR TITLE
Collections images update

### PR DESCRIPTION
**Pull request by mikeger**
# Improvements

- Fixed name accent color for violet color
- Added actions toolbar to swiping images controller. Now you can save/forward/copy/reveal image from the toolbar.
- Unified message actions handling.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Collections/CollectionsViewController.swift [hunk 1]
```diff
@@ -549,6 +549,8 @@ extension CollectionsViewController: MessageActionResponder {
         if Message.isImageMessage(message) {
             switch action {
             case .forward: fallthrough
+            case .copy: fallthrough
+            case .save: fallthrough
             case .showInConversation:
```
Comment (vytis) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionsViewController.swift]: A bit more readable when it's written as ` case .forward, .copy, .save, .showInConversation`

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionsViewController.swift]: But less old-school 🎩 
Will update

Comment (vytis) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionsViewController.swift]: :neckbeard:

**Comment by mikeger** [comment]
@vytis updated

**Pull request merged**